config.md

```diff
@@ -2,11 +2,15 @@
 
 The container's top-level directory MUST contain a configuration file called `config.json`.
 The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go).
+Platform-specific configuration schema are defined in [**`platform-specific documents`**](#platform-specific-configuration) linked below.
```

Comment:
We probably don't need backticks or bold around “platform-specific documents”.  The usual link styling makes the link-ness obvious enough.